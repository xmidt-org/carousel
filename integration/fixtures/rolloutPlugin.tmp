terraform init
terraform apply --auto-approve -var versionGreen=0.1.0 -var versionGreenCount=2