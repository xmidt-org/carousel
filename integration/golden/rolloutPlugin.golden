{
  "blue": {
    "count": 3,
    "version": "0.3.1"
  },
  "green": {
    "count": 0,
    "version": "0.1.0"
  }
}
